allure-web/src/plugins/timeline/TimelineView.js

```diff
@@ -47,45 +48,28 @@ class TimelineView extends BaseChartView {
 
     setupViewport() {
         var selectedPercent = (100*this.data.selectedTestCases/this.model.get('statistic').total).toFixed(2);
-        this.$el.html(`<div class="timeline__header">
-            <span class="timeline__header__min_duration">${duration(this.minDuration)}</span>
-            <span class="timeline__current_duration">
-                Selected ${this.data.selectedTestCases}(${selectedPercent}%) tests
-                with duration above ${duration(this.selectedDuration|0, 2)}
-            </span>
-            <span class="timeline__header__max_duration">${duration(this.maxDuration, 2)}</span>
-            <input class="timeline__range_bar"
-                type="range"
-                min=${this.minDuration}
-                max=${this.maxDuration}
-                value=${this.selectedDuration}>
-            </input>
-        </div>
-        <div class="timeline__body">
-            <div class='timeline__chart'>
-                <svg class="timeline__chart_svg">
-                    <g class="timeline__chart__axis timeline__chart__axis_x"></g>
-                    <g class="timeline__plot"
-                        transform="translate(0, ${4*BAR_GAP})">
-                    </g>
-                </svg>
-            </div>
-            <div class='timeline__brush'>
-                <svg class="timeline__brush_svg">
-                    <rect class="timeline__brush__axis_bg"
-                        width="${this.$el.width()}"
-                        height="${PADDING}"
-                        fill="white"
-                        transform="translate(0, ${BRUSH_HEIGHT + BAR_GAP})">
-                    </rect>
-                    <g class="timeline__brush__axis timeline__brush__axis_x"></g>
-                </svg>
-            </div>
-        </div>`);
+        this.$el.html(template({
+            minDuration: this.minDuration,
+            selectedTestCases: this.data.selectedTestCases,
+            selectedPercent: selectedPercent,
+            selectedDuration: this.selectedDuration|0,
```

Comment:
This is bitwise operator. Did you mean `selectedDuration: this.selectedDuration || 0,`